Library/.rubocop.yml

```diff
@@ -27,6 +27,9 @@ FormulaAudit/Options:
 FormulaAuditStrict/Options:
   Enabled: true
 
+NewFormulaAudit/Options:
+  Enabled: true
```

Comment:
@GauthamGoli I've disabled this in https://github.com/Homebrew/brew/commit/207ffd6d4f404c4093efbf24c19650730fbb27e4 as it was activating for every formula every time.